Robin/HW1_Robin/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="mock.css">
+		<meta charset="utf-8">
+		<title>Orange Udacity Mug</title>
+	</head>
+	<body>
+
+		<h1>Orange Udacity Mugs</h1>
+		
+		<div class="sidebar">
+			<img src="asset/mug.png" alt="Orange Mug">
+			<div class="features">
+				<h2>Main Features</h2>
+				<ul class='list'>
+					<li>Holds your favorite drink</li>
+					<li>Energizing color</li>
+					<li>Reminds you to learn</li>
+				</ul>
+			</div>
+		</div>
+
+		<main><article>
```

Comment:
I used main here because I this is the main content of a document. And article tag here to remind Web crawler that this is an article.